droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/action/CommandFactoryImpl.java

```diff
@@ -378,6 +383,11 @@ private void processCommandLineCSVOptions(CommandLine cli, PropertiesConfigurati
         if (cli.hasOption(CommandLineParam.QUOTE_COMMAS.getLongName())) {
```

Comment:
This private method can be renamed to something like "processCommandLineExportOptions" to remove `csv` from the name